ui/app/templates/components/job-page/parts/title.hbs

```diff
@@ -33,7 +38,13 @@
         @confirmText="Yes, Purge Job"
         @confirmationMessage="Are you sure? You cannot undo this action."
         @awaitingConfirmation={{this.purgeJob.isRunning}}
-        @onConfirm={{perform this.purgeJob}} />
+        @onConfirm={{perform this.purgeJob}}
+        {{keyboard-shortcut 
+          label="Exec"
+          pattern=(array "p" "u" "r" "g" "e")
+          action=(perform this.purgeJob)
+        }}
```

Comment:
It's a pretty long shortcut, but I'm kind of wary about the potential for unintended destructive operations here. Would it be possible to still have the confirmation pop-up? Keyboard shortcuts make things quick but sometimes it's good to slow down and double-check you are in the right namespace or something like that 😅

Reply:
I've been giving this some thought. A couple things:
- Our two-step buttons are probably not a convention that we're going to continue with in the near future; [Helios](https://helios.hashicorp.design/components/modal) indicates that we should either use modals to warn, or toast/modals to make undoing easy. Once we start using modals, adding them to a queue of keyboard shortcut actions will be possible, but still not an ideal flow. As it stands now, the visual effect probably wouldn't be enough to make the user notice and they might have assumed the purge command went through without noticing otherwise.
- The act of typing the full word is a preeeeeetty deliberate action - something much harder to do than accidentally mouse-clicking or tab+entering on the button while enroute to do something else on the page.

As such, I think I'd like to move forward with this change. I am worried about the "double-checking you're in the right namespace" problem as well but the benefit here seems to outweigh the potential cost.